.changeset/thirty-days-tell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-backend-module-pg': patch
+---
+
+Support pg client customize analyzer
```

Comment:
I'm not sure where this would go but we should include some documentation about how to use this or if it's something purely done on the PG side maybe we point to their documentation?

Reply:
sure

Reply:
sure